add: harshhere

harshhere905 opened the pull request:
## Description

[Include a brief description of the changes you've made]

## Related Issues

[Include any related issues or pull requests that this PR addresses or is related to. Use GitHub's shorthand syntax to link to them, like #1234.]

## Changes Proposed

[Describe the changes you've made in detail. Be specific and include any relevant code snippets.]

## Checklist

- [x] I have read and followed the [Contribution Guidelines](https://github.com/shyamtawli/devFind/blob/master/CONTRIBUTING.md).
- [x] All new and existing tests passed.
- [x] I have updated the documentation to reflect the changes I've made.
- [x] My code follows the code style of this project.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

## Note to reviewers

shyamtawli reviewed (changes requested):
@harshhere905 You have to also add the file name that is `Priyanshi.json` in ProfileList.json file. 

Check the steps that are provided in readme.

harshhere905 commented:
@shyamtawli  I have added the file name that is Priyanshu.json in ProfileList.json file #531.. Also added  my avatar tag in the file. now you can merge my repo

shyamtawli commented:
Have you opened a new PR for it? You can commit changes to this PR only.

harshhere905 commented:
@shyamtawli I have commited the changes to this pr. You can consider it to merge.. thanks

shyamtawli commented on public/data/Priyanshu.json:
```diff
@@ -0,0 +1,12 @@
+  "name": "Priyanshu Varshney",
+  "location": "India",
+  "bio": "A Web Programmer",
+  "avatar": "https://avatars.githubusercontent.com/u/124420199?s=400&u=fceacb048a62263a94621c775c13754b8dbc9bd0&v=4"
```
```suggestion
  "avatar": "https://avatars.githubusercontent.com/u/124420199?s=400&u=fceacb048a62263a94621c775c13754b8dbc9bd0&v=4",
```
Add comma at last after avatar tag.

harshhere905 commented:
@shyamtawli done!

shyamtawli commented:
@harshhere905 Thank you so much for your contribution! Your efforts are greatly appreciated and will go a long way in improving our project. Please feel free to share it with others and help us grow even further!
